src/mongoc/mongoc-write-concern.c

```diff
@@ -98,7 +100,7 @@ bool
 mongoc_write_concern_get_fsync (const mongoc_write_concern_t *write_concern)
 {
    bson_return_val_if_fail(write_concern, false);
-   return write_concern->fsync_;
+   return (write_concern->fsync_ == true);
```

Comment:
@hanumantmk: since the setter functions use bool arguments, are we guaranteed that only `true` or `false` can ultimately be assigned? I modified the tests to assign arbitrary integers and that does seem to be the case -- I just wasn't expecting the compiler to do that for us.